README.md

```diff
@@ -2,9 +2,84 @@
+  * Entropy-stable shock capturing
+  * Positivity-preserving limiting
+  * Compatible with the [SciML ecosystem for ordinary differential equations](https://diffeq.sciml.ai/latest/)
+  * CFL-based and error-based time step control
+* Shallow water capabilities
+  * Wetting and drying
+  * Multi-layer flows
+  * Sediment transport via an Exner model
+
+## Examples
+
+## Installation
+If you have not yet installed Julia, please [follow the instructions for your
+operating system](https://julialang.org/downloads/platform/). TrixiShallowWater.jl works
+with Julia v1.8 and newer. We recommend using the latest stable release of Julia.
+
+[comment]: <> (We can update this with a "for users" and "for developers" section once the package is registered)
+
+TrixiShallowWater.jl is **not** a registered Julia package, and therefore needs to be downloaded manually and then run from with the cloned directory:
+```bash
+git clone https://github.com/trixi-framework/TrixiShallowWater.jl.git
+julia --project=@.
+```
+In addition TrixiShallowWater.jl requires the numerical solver framework [Trixi.jl](https://github.com/trixi-framework/Trixi.jl), [OrdinaryDiffEq.jl](https://github.com/SciML/OrdinaryDiffEq.jl) for time integration, and [Plots.jl](https://github.com/JuliaPlots/Plots.jl) for visualization, which can be installed by executing the following in the Julia REPL:
+```julia
+julia> using Pkg
+
+julia> Pkg.add(["Trixi", "Trixi2Vtk", "OrdinaryDiffEq", "Plots"])
+```
+
+## Authors
+TrixiShallowWater.jl is maintained by the
+[Trixi authors](https://github.com/trixi-framework/Trixi.jl/blob/main/AUTHORS.md).
+Its principal developers are [Andrew Winters](https://liu.se/en/employee/andwi94)
+(Linköping University, Sweden)
+and [Patrick Ersing](https://liu.se/en/employee/pater53)
+(Linköping University, Sweden).
+The full list of contributors can be found in [AUTHORS.md](AUTHORS.md).
+
+## License and contributing
+TrixiShallowWater.jl is licensed under the MIT license (see [LICENSE.md](LICENSE.md)).
+Since TrixiShallowWater.jl is an open-source project, we are very happy to accept contributions from the
+community. To get in touch with the developers,
+[join us on Slack](https://join.slack.com/t/trixi-framework/shared_invite/zt-sgkc6ppw-6OXJqZAD5SPjBYqLd8MU~g)
+or [create an issue](https://github.com/trixi-framework/TrixiShallowWater.jl/issues/new).
+
+## Acknowledgments
+<p align="center" style="font-size:0;"><!--
+  SRC      --><img align="middle" src="https://github.com/trixi-framework/Trixi.jl/assets/3637659/48f9da06-6f7a-4586-b23e-739bee3901c0" height="120"><!--
+  -->
+</p>
+
+This project has benefited from funding from [Vetenskapsrådet](https://www.vr.se)
+(VR, Swedish Research Council), Sweden
+through the VR Starting Grant "Shallow water flows including sediment transport and morphodynamics",
+VR grant agreement 2020-03642 VR.
 
 **Note: This repository is still in its alpha stage and anything might change at
```

Comment:
This part should be removed

Reply:
Do you refer to the note about being in alpha stage or the acknowledgements?

Reply:
The alpha stage. Most parts should be relatively stable as far as I have observed, and the remaining breaking changes can be reflected by new version numbers. This should at least be the case when the package is kind of "published officially".

Reply:
Definitely, I agree. We can ditch this alpha stage note. We just got confused whether or not you meant the statement acknowledging VR.

Reply:
Sorry - I couldn't comment only on the alpha stage part since it was not changed directly in this PR 🙈